modules/node_modules/@ncigdc/containers/FrequentlyMutatedGenesTable.js

```diff
@@ -145,6 +146,11 @@ const FrequentlyMutatedGenesTableComponent = compose(
             ),
           },
           {
+            key: 'annotations',
+            title: 'Annotations',
+            style: { padding: '5px 0 0 0', textAlign: 'center' },
```

Comment:
Can we reference implementation from [react-bootstrap-table](https://github.com/AllenFang/react-bootstrap-table)? Their API mostly seems really elegant and sensible